dashboard/src/components/Common/TextField/TextField.jsx

```diff
@@ -30,6 +30,7 @@ class TextField extends React.Component {
         {this.props.title && <FieldLabel className={styles.title}>{this.props.title}</FieldLabel>}
         <input className='form-control'
           type={this.state.type}
+          name={this.props.fieldProps.name}
```

Comment:
Is this necessary with the `{...fieldProps}` at the end of this component instance?